uaclient/security_status.py

```diff
@@ -159,6 +183,21 @@ def filter_security_updates(
                             (version, expected_origin.site)
                         )
 
+            # This loop should be only used if the user is unattached
```

Comment:
If the user is attached but the services are disabled, this is still valid. So I would update the comments/test names to reflect that.